arm/Microsoft.DataFactory/factories/integrationRuntime/readme.md

```diff
@@ -0,0 +1,42 @@
+# DataFactoryIntegrationRunTimes `[Microsoft.DataFactory/factories/integrationRuntimes]`
+
+## Resource types
+
+| Resource Type | API Version |
+| :-- | :-- |
+| `Microsoft.DataFactory/factories/integrationRuntimes` | 2018-06-01 |
+
+## Parameters
+
+| Parameter Name | Type | Default Value | Possible Values | Description |
+| :-- | :-- | :-- | :-- | :-- |
+| `cuaId` | string |  |  | Optional. Customer Usage Attribution ID (GUID). This GUID must be previously registered|
+| `name` | string |  |  | Required. The name of the Integration Runtime. |
+| `dataFactoryName` | string |  |  | Required. The name of the Azure Factory to create |
+| `managedVirtualNetworkName` | string |  |  | Required. The name of the Managed Virtual Network. |
+| `typeProperties` | object |  |  | Required. Managed integration runtime type properties. |
+
+### Parameter Usage: `typeProperties`
+
```

Comment:
This is a very complicated object. Please add this [reference](https://docs.microsoft.com/en-us/azure/templates/microsoft.datafactory/factories/integrationruntimes?tabs=bicep#integrationruntime-objects) here:

Reply:
Added!